bin/cml/runner.js

```diff
@@ -47,12 +47,19 @@ const shutdown = async (opts) => {
 
   const retryWorkflows = async () => {
     try {
-      if (!noRetry && RUNNER_JOBS_RUNNING.length) {
-        await Promise.all(
-          RUNNER_JOBS_RUNNING.map(
-            async (job) => await cml.pipelineRestart({ jobId: job.id })
-          )
-        );
+      if (!noRetry) {
+        if (cml.driver === 'github') {
+          const job = await cml.runnerJob({ runnerId: RUNNER_ID });
+          if (job) RUNNER_JOBS_RUNNING = [job];
+        }
+
+        if (RUNNER_JOBS_RUNNING.length) {
```

Comment:
I take it you guys are okay with this style of truthy-ness? On a broader scope, do you reference anything in particular for js/node style guide?

Reply:
@dacbd I do not follow you 🙂 could you please elaborate a bit more?

Reply:
Sure, being more explicit in the condition `if (RUNNER_JOBS_RUNNING.length)` the logical question is "Are there jobs in the array?" I suppose my question is, is there an organizational style guide on truthy js values. 
```
console.log(
  -1 ? "true" : "false",
  0 ? "true" : "false",
  1 ? "true" : "false"
)
```

Reply:
> is there an organizational style guide on truthy js values

organizational at Iterative? We have an eslint in place . We could have done explicit comparison but JS behaves equally with one additional benefit: array.length is fastest in speed (and shorter in writing) than array.length > 0 

```js
if (RUNNER_JOBS_RUNNING.length > 0)
```

What is the point? Are you interested in learning out JS style? 🙂

Reply:
> _I take it you guys are okay with this style of truthy-ness?_

Technically speaking, yes, “<i><a href="https://262.ecma-international.org/5.1/#sec-15.4">every Array object has a length property whose value is always a <b>nonnegative</b> integer less than 2<sup>32</sup></a></i>”

> _On a broader scope, do you reference anything in particular for js/node style guide?_

Supposedly (#596), we follow the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript), but the conditional you've spotted is a documented antipattern:

> _[15.3 Use shortcuts for booleans, but explicit comparisons for strings and numbers.](https://github.com/airbnb/javascript#comparison--shortcuts)_
>
> ```javascript
> // bad
> if (collection.length) {
>   // ...
> }
> 
> // good
> if (collection.length > 0) {
>   // ...
> }
> ```

Reply:
> Supposedly (https://github.com/iterative/cml/issues/596)

Its still open. We never stick to it yet aside having the original Iterative linter. As far as I remember

Reply:
https://cml.dev/doc/contributing/core#submitting-contributions
> _All JS code should follow the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript)._

Reply:
> should

should is the keyword here

Reply:
```py
# best
if collection:
    print("the zen of Python")
```

Reply:
Maybe I have found the why we are delayed 😁

Reply:
I have fix it to not have to debate philosophically during the next 2 hours 😁

Reply:
> What is the point? Are you interested in learning our JS style? 🙂 

Yes